.github/resources/.sensitive-files

```diff
@@ -5,3 +5,4 @@ ops/terraform/env/mgmt/base_config/values/mgmt.yaml
 ops/terraform/services/base/values/prod-sbx.yaml
 ops/terraform/services/base/values/prod.yaml
 ops/terraform/services/base/values/test.yaml
+ops/terraform/services/base/values/ephemeral.yaml
```

Comment:
Totally reasonable. Could you fix your editor configuration? Maybe https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
Thanks, fixed and added that to my VS Code config 👍